gyp/.github/workflows/release-please.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   release-please:
     runs-on: ubuntu-latest
     steps:
-      - uses: google-github-actions/release-please-action@v3
+      - uses: googleapis/release-please-action@v3
```

Comment:
Changes in `gyp/` should be submitted to https://github.com/nodejs/gyp-next instead.